src/index.ts

```diff
@@ -72,6 +73,7 @@ export class AdgoError extends Error {
         this.code = (error as AdgoError).code || (data.code as string) || code;
         this.name = (error as AdgoError).name || 'AdgoError';
         this.data = merge({}, (error as AdgoError).data, data);
+        this.sentry = merge({ sentry: true }, error as AdgoError, data).sentry;
```

Comment:
I understand what we are doing, but man is it confusing that we set all these fields in data as well as class attributes